src/Schema/Events/Tap.ts

```diff
@@ -647,3 +647,31 @@ export interface TappedVerifyIdentity {
   sale_id: string
   subject: string
 }
+
+/**
+ * A user taps on any hyperlinked text
+ *
+ * This schema describes events sent to Segment from [[tappedLink]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedLink",
+ *    context_module: "inboxActiveBids",
+ *    context_screen_owner_type: "inboxBids",
+ *    destination_screen_owner_slug: "/favorites"
+ *    external: false
+ *  }
+ * ```
+ */
+export interface TappedLink {
+  action: ActionType.tappedLink
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  destination_screen_owner_type?: ScreenOwnerType
+  destination_screen_owner_id?: string
+  destination_screen_owner_slug: string
```

Comment:
i think maybe `destination_screen_owner_slug` is optional too, unless you're referring to `destination_path` -- which I believe we use as a property in other events. `destination_path` works for internal or external links, where external links won't have a `slug` that corresponds to some artsy data object (like an artwork, artist, or partner).

Reply:
tl;dr:
- suggesting `destination_path` as required prop
- suggesting `destination_screen_owner_slug?`

Reply:
Correct, I'd agree with that 👍

Reply:
Ok thanks